1.18.2

matteobruni opened the pull request.

matteobruni committed:
build: fixes autocomplete on vscode, fixes #922

matteobruni committed:
build: prettified code

matteobruni committed:
build: packages updated

matteobruni committed:
build: fixed build for react demo

matteobruni committed:
build: added JSDoc to preact and inferno too

matteobruni committed:
Merge branch 'master' into dev

matteobruni committed:
build: updated snow preset

matteobruni committed:
build: fixed warnings on inferno

ghost commented:
<img src="https://www.deepcode.ai/icons/green_check.svg" width= "50px" align= "left"/> Congratulations :tada:. DeepCode [analyzed](https://www.deepcode.ai/app/gh/matteobruni/tsparticles/56e51c918f51ff10e6e0200d4476a17aabe8d7ed/matteobruni/tsparticles/7cbfa46a827266e5c4186420ca2660c500767334/pr/_/%2F/code/?utm_source=gh_review&c=0&w=0&i=0&) your code in 3.209 seconds and we found no issues. Enjoy a moment of no bugs :sunny:.

#### 👉 View analysis in [**DeepCode’s Dashboard**](https://www.deepcode.ai/app/gh/matteobruni/tsparticles/56e51c918f51ff10e6e0200d4476a17aabe8d7ed/matteobruni/tsparticles/7cbfa46a827266e5c4186420ca2660c500767334/pr/_/%2F/code/?utm_source=gh_review&c=0&w=0&i=0&) | [_Configure the bot_](https://www.deepcode.ai/app/gh/?ownerconfig=matteobruni)

matteobruni committed:
chore(release): published new version

 - ng-particles@2.1.2
 - inferno-particles@1.1.2
 - jquery-particles@1.18.2
 - preact-particles@1.18.2
 - react-tsparticles@1.18.2
 - svelte-particles@1.1.2
 - particles.vue@2.1.2
 - particles.vue3@1.1.2
 - tsparticles-editor@1.18.2
 - tsparticles@1.18.2
 - ng-particles-demo@1.3.2
 - editor-demo@1.3.2
 - inferno-particles-demo@1.1.2
 - jquery-particles-demo@1.3.2
 - tsparticles-demo@1.3.2
 - preact-particles-demo@1.3.2
 - react-tsparticles-demo@1.3.2
 - svelte-demo@1.1.2
 - particles.vue-demo@1.3.2
 - particles.vue3-demo@1.1.2
 - tsparticles-preset-60fps@1.3.2
 - tsparticles-preset-background-mask@1.3.2
 - tsparticles-preset-basic@1.3.2
 - tsparticles-preset-big-circles@1.3.2
 - tsparticles-preset-bouncing@1.3.2
 - tsparticles-preset-fire@1.3.2
 - tsparticles-preset-font-awesome@1.3.2
 - tsparticles-preset-fountain@1.3.2
 - tsparticles-preset-snow@1.3.2
 - tsparticles-preset-stars@1.3.2
 - tsparticles-shape-bubble@1.3.2
 - tsparticles-shape-heart@1.3.2
 - tsparticles-shape-multiline-text@1.3.2
 - tsparticles-shape-spiral@1.3.2
 - cra-template-particles-typescript@1.1.2
 - cra-template-particles@1.1.2
